.github/workflows/builds.yml

```diff
@@ -1,40 +1,47 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Build tests
 
-# Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
+  release:
+
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
+    inputs:
+      build_release:
+        description: 'Build Release binaries'
+        required: false
+        type: boolean
 
 # Set the build type here
 env:
-  CI_ENV_BUILD_TYPE: Debug
-  B_BUILD_DIR: build
-  B_BUILD_TYPE: Debug
-  INPUTLEAP_VERSION_STAGE: Debug
+  B_BUILD_TYPE: ${{ (inputs.build_release || startsWith(github.ref, 'refs/tags/v')) && 'Release' || 'Debug' }}
```

Comment:
Is it possible to check if `build_release` is set at all? If yes, I think it makes sense to override `refs/tags/v` which whatever value that is set by `build_release` variable.

Reply:
Currently `build_release` is a boolean.

The logic here is build `Release` if the build is triggered by a release (triggered event is a tag, and tag starts with `v`) or `Release` checkbox is ticked.

In any case it’s the version defined in the code that is used for the build.

Reply:
The logic can be adjusted of course. Just the manual setting? Just on tags pushed? Something else?

Reply:
I like the existing solution, personally, and don't want to override any `refs/tags/v`. @Cimbali's solution makes sense from a CI/CD perspective, and is logical.

Reply:
Sorry for the delay. @shymega this basically means that it's not possible to create a debug build on v1.2.3 tag for example. Do we really want that? I think it makes sense to leave it as an option.

Reply:
I’ll double check, but I believe you can still do that, by triggering a build on a tag manually (so the condition “triggered event is a tag” is false) and leaving the “release” checkbox unticked.

Reply:
@p12tic Right. Well, let's wait for @Cimbali to check, and then discuss further after they come back to us.

Reply:
@Cimbali Did you get a chance to double check?

cc: @p12tic